members/Ganesh0403.md

```diff
@@ -0,0 +1 @@
+Ganesh Shinde - shindeg0304@gmail.com - @Ganesh0403 - Pimpri Chinchwad College of Engineering, Pune - Computer
```

Comment:
Username should be linked to your github user account.

Like this - `[@Ganesh0403](https://github.com/Ganesh0403)`